tests/posts_tests/test_search_views.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.django_db
+class TestViews:
+
+    search_url = '/search/?searched='
+
+    def test_searching_username_redirection_view(self, client, persist_user, persist_post):
```

Comment:
Redirect usually means that the response status code is 3xx (302 for example).
I see here only a status code of 200, maybe worth to change the test name?

Reply:
fixed